literature.qmd

```diff
@@ -0,0 +1,7 @@
+---
+title: "Literature"
+---
+
+This is a collection of references to empirical studies, conceptual articles about LLMs in education, and other publications relevant to understanding the current moment. This is a snapshot (updated 5/25/23) of a [Zotero group](https://www.zotero.org/groups/5034711/llms_for_edu); read/write access available on request if you have a suggestion or would like to see the latest & greatest.
+
+This is a work in progress; a current formatted version of the literature is available [here](http:resources/annotated-bibliography.html)
```

Comment:
@christopherormerod the URL for the formatted lit review is to ah HTML file created external to QMD, then posted here - I'm making an error in how I reference it, in case you have an idea to fix it.  I can't figure out how to render out the Zotero group into a file that is native QMD without entering a citation for every item, which seems absurd.